components/Header.tsx

```diff
@@ -32,7 +32,7 @@ function Header() {
 
           <div className="flex gap-2 lg:ml-40">
             <button
-              className="btn btn-ghost capitalize"
+              className={`relative  bg-2023-void-2 hover:bg-2023-void-2 btn  text-white rounded-1xl  hover:text-shadow hover:shadow-2023-bavarian-gold-3 `}
```

Comment:
Can you just switch back to normal string instead of the template literal?

Reply:
Also, the border of the button isn't that visible. Can you also fix that?
![image](https://github.com/max-programming/hacktoberfest-projects/assets/51731966/71e86919-81ba-4f84-9831-e0d13b350ed0)

Reply:
@max-programming  Yes I can Do it

Reply:
@max-programming  Can you please Suggest this is okay or not ?
![image](https://github.com/max-programming/hacktoberfest-projects/assets/87563885/c01530d8-3654-42f6-ab0c-7113119dfeed)

Reply:
@Dhruvn-patel The previous design was better. Just make the border lighter so that it's visible